worker/src/main.rs

```diff
@@ -278,10 +264,9 @@ fn worker(
 
     let handle = TOKIO_HANDLE.lock().unwrap().as_ref().unwrap().clone();
     handle.spawn(async move {
-        substratee_worker_rpc_server::run_server(
-            &url,
-            enclave,
-        ).await.unwrap()
+        substratee_worker_rpc_server::run_server(&url, enclave)
```

Comment:
I believe we need to unify our rustfmt :)

Reply:
I was told to use these settings (using rustfmt, not the clion built-in formatter)?:

![rustfmt_setting_clion](https://user-images.githubusercontent.com/9334190/124605914-afab3780-de6c-11eb-84ce-27d4e8de8d46.PNG)

Reply:
haha thought I enabled that, apparently I haven't.  Probably disabled it because a lot of our code is not formatted along this guideline. Maybe we should do an fmt PR soon.

Reply:
no worries 😄 I'm glad we're synced up now with this matter. As for the separate PR for formatting changes: We could do that, we just have to make sure that as little as possible gets developed in parallel, since merge conflicts will be guaranteed. That's why in other projects where we faced this problem, we went the route of just formatting as we go along (especially in a large team)

Reply:
However, substrate does not use rustfmt currently. In the node I disable rustfmt completely because when we update deps, we also need to align some code, which I usually do with a directory diff.

Reply:
okay, that's good to know - fortunate for me that I haven't worked in the node yet 😅